lib/statsd-client.js

```diff
@@ -71,6 +71,22 @@ StatsDClient.prototype.increment = function (name, delta) {
 };
 
 /*
+ * counterBeforeTermination(name, delta)
+ */
+StatsDClient.prototype.immediateCounter = function (name, delta, cb) {
```

Comment:
For consistency; please add immediateGauge and immediateTimer.

Also add tests for these methods.